### ISSUE TITLE
add ability to transcode video using the AV1 codec

### ISSUE THREAD

**Issue by oupala** [thumbsup/thumbsup#148]
## Feature request

AV1 is a new codec which is far better than VP9 and HEVC, as written in the [wiki of mmpeg](https://trac.ffmpeg.org/wiki/Encode/AV1):

> libaom-av1 is the AOMedia video encoder for ​AV1, an open source & royalty-free video codec. libaom-av1 can save about 30% bitrate compared to VP9 and H.265 / HEVC, and about 50% over H.264, while retaining the same visual quality.

AV1 is already available in the next debian stable release: buster. It will probably soon be available in all other major linux distribution. VLC 3 and ffmpg 4 are already supporting it.

THe AV1 codec is available with [mp4 or webm containers](https://en.wikipedia.org/wiki/AV1#Operating_system_support) (and also mkv containers, but mkv containers are not widely used on the web).

AS mentionned in #130, at some point, it would be a good idea to change the codec of thumbsup to AV1, whatever container (format) is set on thumbsup configuration (mp4 or webm).

**Comment by oupala** [comment]
AV1 is gaining experience and start to be usable on a day-to-day basis.

**Comment by rprieto** [comment]
Thanks @oupala. What do you think of a new option, such as

```bash
  --video-format        Video output format  [choices: "mp4", "webm"] [default: "mp4"]
  --video-codec         Video codec  [choices: "h265", "av1"] [default: "h265"]
```

You're right this will require a change in the Downsize module, which should be easy to test in isolation.

**Comment by oupala** [comment]
This is the right decision.

But maybe `video format` could be renamed to `video container`. `video container` is the right term to use, nonetheless it is less known than `video format`. Hence, not sure what term is the most appropriate: the right one, or the more widely known.

**Comment by rprieto** [comment]
You're right it would be more correct. However I think I'll keep `--video-format` as it will be more easily understood by everyone and also because it's an existing flag (no breaking change).
